sdk/eventhubs/azure-messaging-eventhubs-stress/workbooks/details.json

```diff
@@ -0,0 +1,290 @@
+        "size": 0,
+        "showAnalytics": true,
+        "title": "Test results",
+        "noDataMessageStyle": 5,
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components",
+        "sortBy": []
+      },
+      "customWidth": "100",
+      "name": "query - 9"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "// rate\r\nlet runId = \"{runId}\";\r\nlet role_name=strcat(\"java-eventhubs-\", runId);\r\nlet send = customMetrics\r\n| where timestamp >= {timeRange:start} and timestamp <= {timeRange:end}\r\n| where name == \"messaging.eventhubs.events.sent\" and cloud_RoleName == role_name and cloud_RoleInstance==\"sender\"\r\n| where customDimensions[\"otel.status_code\"] == \"ok\" or customDimensions[\"otel.status_code\"] == \"\"\r\n| summarize message_send=sum(valueSum) by bin(timestamp, 1m);\r\nlet receive = customMetrics\r\n| where timestamp >= {timeRange:start} and timestamp <= {timeRange:end}\r\n| where name == \"messaging.eventhubs.consumer.lag\"\r\n| where cloud_RoleName == role_name and cloud_RoleInstance==\"receiver\"\r\n| summarize receive = sum(valueCount) by bin(timestamp, 1m);\r\nlet settle = customMetrics\r\n| where timestamp >= {timeRange:start} and timestamp <= {timeRange:end}\r\n| where name == \"messaging.eventhubs.checkpoint.duration\"\r\n| where cloud_RoleName == role_name and cloud_RoleInstance==\"receiver\"\r\n| where customDimensions[\"otel.status_code\"] == \"ok\" or customDimensions[\"otel.status_code\"] == \"\"\r\n| summarize settle = sum(valueCount) by bin(timestamp, 1m);\r\nsend \r\n| join kind = fullouter (receive) on timestamp\r\n| join kind = fullouter (settle) on timestamp\r\n| project-away timestamp1, timestamp2\r\n| render timechart",
+        "size": 0,
+        "aggregation": 3,
+        "title": "Send/receive/settle success rate (per minute)",
```

Comment:
wondering what is `settle` here mean, is it checkpointing?

Reply:
good catch, yes, will change